code/_macros_arfs.dm

```diff
@@ -0,0 +1,4 @@
+//bayadd
+#define JOINTEXT(X) jointext(X, null)
```

Comment:
Why is this a define?

Reply:
circuits

Reply:
Why not use the jointext proc directly?

Reply:
Circuitry needs this version, otherwise it breaks.